docs/04-line.md

```diff
@@ -44,3 +44,4 @@ const lineChart = new chartXkcd.Line(svg, {
     - up right: `chart.Xkcd.positionType.upLeft`
 - `dataColors`: array of colors for different datasets
 - `fontFamily`: customize font family used in the chart
+- `disableEffect`: disable xkcd effect for serious people (default `false`)
```

Comment:
About the option name, `unxkcdify` sounds better to me. What do you think?

Reply:
I don't mind. If you wish so I will change the option name. 
Maybe `unXkcdify` to respect camelCase?

Reply:
`unxkcdify` should work, like `uncomment` it is one word

Reply:
Updated